plugin/move-less.vim

```diff
@@ -117,7 +134,7 @@ function! MoveLessMode()
             endif
             exec "normal! z."
             redraw
-        elseif l:result ==# 'h' || l:result ==# 'L'
+        elseif l:result ==# g:MoveLess#Mappings['UnfoldAboveAndBelowK1'] || l:result ==# g:MoveLess#Mappings['FoldAboveAndBelowK2']
```

Comment:
i guess here is an error, shouldn't it be `UnfoldAboveAndBelowK2` for the second key?

Reply:
Ugh, I think I should have gone to the first map iteration and redone the substitution manually instead of copying it over... I'll double check in the next commit